experimental/skills/DialogToDialog/DialogRootBot/README.md

```diff
@@ -0,0 +1,21 @@
+﻿# DialogRootBot (**DRAFT**)
```

Comment:
I can copy edit or help with a draft of the README offline.

Reply:
That would be great, thanks! 

---
In reply to: [356238520](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356238520) [](ancestors = 356238520)